### ISSUE TITLE
input.mergeMap is not a function

### ISSUE THREAD

**Issue by douglas168** [creativetimofficial/material-dashboard-pro-angular#164]
Hi, 

I purchased 'Material Dashboard Pro Angular' and my angular CLI is 1.6.3'.

When I run 'ng g module newModule' at the project folder, I get 

input.mergeMap is not a function

What could be the problem and how do fix it?

Thanks,
Douglas

**Comment by tusharadha** [comment]
Hi,

I have the same problem.  I have tried many solutions but none worked so far.  

Any update on this Creative tim?

Thanks

**Comment by tusharadha** [comment]
Finally I was able to solve this issue. 
Install devkit schematics.  I have installed 0.0.34 as it is compatible with angular cli 1.4.2.  

`npm i @angular-devkit/schematics@0.0.34 --save`

if your global version of angular cli is higher I recommend you to downgrade it to the version specified into your local "`package.json`" file.

`npm remove -g @angular/cli`
and install the 1.4.2 version
`npm i -g @angular/cli@1.4.2`

Thanks,
